INSTALL.md

```diff
@@ -7,116 +7,135 @@
 Installing OpenCoarrays
 =======================
 
-[![Download as PDF][pdf img]](http://md2pdf.herokuapp.com/sourceryinstitute/opencoarrays/blob/master/INSTALL.pdf)
+[The following link is commented until it can be updated. To uncomment delete the backslashes, the ourtermost square brackets, and the trailing colon and hash mark]:#
+[\[\!\[Download as PDF]\][pdf img]](http://md2pdf.herokuapp.com/sourceryinstitute/opencoarrays/blob/master/INSTALL.pdf)]:#
 
 Download this file as a PDF document
 [here](http://md2pdf.herokuapp.com/sourceryinstitute/opencoarrays/blob/master/INSTALL.pdf).
 
  *  [End-User Installation]
-     * [Installation Script]
-     * [OS X]
+     * [macOS]
      * [Windows]
      * [Linux]
+     * [Virtual machine]
+     * [Installation Script]
  *  [Advanced Installation from Source]
      *  [Prerequisites]
-     *  [CMake]
+     *  [CMake scripts]
      *  [Make]
  *  [Obtaining GCC, MPICH, and CMake]
 
 End-User Installation
 ---------------------
 
-### Installation Script###
+Most users will find it easiest and fastest to use package management software to install OpenCoarrays.
+Package management options for macOS (formerly known as OS X), Windows, and Linux are described first 
+below. Also described below are options for installing via the Sourcery Institute virtual machine or
+the OpenCoarrays installation script.
+
+### macOS ###
 
-As of release 1.2.0, users might consider installing by downloading and uncompressing
-a file from our [Releases] page and running the installation script in the top-level
-source directory:
+OS X users may use the [Homebrew] or [MacPorts] package management systems to install OpenCoarrays.
```

Comment:
Can you please add a note that Homebrew is the preferred method? I should be able to get it to the point where our CI/release scripts automatically submit a PR with the updated Homebrew formula to homebrew-core. Then we won't have to continue maintaining MacPorts, unless you really want to...